src/wp-admin/my-sites.php

```diff
@@ -10,11 +10,11 @@
 require_once __DIR__ . '/admin.php';
 
 if ( ! is_multisite() ) {
-	wp_die( __( 'Multisite support is not enabled.' ) );
+	wp_die( __( 'Multisite support is not enabled.' ), 409 );
```

Comment:
Same question here about 501 vs 409?

```suggestion
	wp_die( __( 'Multisite support is not enabled.' ), 501 );
```